server/runtime/src/test/java/org/infinispan/server/test/ContainerInfinispanServerDriver.java

```diff
@@ -56,7 +57,9 @@
    CountdownLatchLoggingConsumer latch;
    ImageFromDockerfile image;
    private File rootDir;
+
    private final boolean preferContainerExposedPorts = Boolean.getBoolean("org.infinispan.test.server.container.preferContainerExposedPorts");
+   private final int serverStartupTimeout = Integer.getInteger("org.infinispan.test.server.container.serverStartupTimeout", 30);
```

Comment:
@diegolovison @tristantarrant I thought our test properties start with `infinispan.test`, without the `org.`

Personally I don't see the utility in making this configurable though, do we really expect in environment where the server takes more than 30s to start the tests can still run ok?